dubbo-configcenter/dubbo-configcenter-zookeeper/src/main/java/org/apache/dubbo/configcenter/support/zookeeper/CacheListener.java

```diff
@@ -38,15 +37,12 @@
  */
 
 public class CacheListener implements DataListener {
```

Comment:
Is CacheListener still needed? Can we connect TargetListener(zookeeper listener) and DataListener(dubbo listener) directly?

Reply:
CacheListener can hold same path listeners, ZookeeperListener : ConfigurationListener -> 1 : N.
We still use CacheListener, ZookeeperListener : CacheListener -> 1 : 1, CacheListener : ConfigurationListener -> 1 : N.

CacheListener is an adapter, it holds real listeners,  and adapt kinds of listener (zookeeper, nacos ...)